### ISSUE TITLE
The expressions seem to come from the reference image instead of the current image

### ISSUE THREAD

**Issue by HongchengZhao** [hologerry/IMF#7]
![individualImage](https://github.com/user-attachments/assets/546deb4c-e626-4569-9552-811da2fef7a0)
The first row is the reference image, and the second row is the current one. And you can see the reconstructed image always has the same expression as the reference image. I added extra L1 loss around eyes and mouth area, but that does not help. Do you meet the same problem? If so, would you please elaborate on how to solve it? Thank you.

**Comment by johndpope** [comment]
Don’t know if it helps - https://github.com/johndpope/IMF
I want to experiment with higher res. 

There’s images successfully recreating here 

https://wandb.ai/snoozie/IMF/runs/xscj3hjo?nw=nwusersnoozie

**Comment by HongchengZhao** [comment]
> Don’t know if it helps - https://github.com/johndpope/IMF I want to experiment with higher res.
> 
> There’s images successfully recreating here
> 
> https://wandb.ai/snoozie/IMF/runs/xscj3hjo?nw=nwusersnoozie

Thanks for your great work, and it does seem to be recreating well. Though I wonder whether this run is overfitted or not?

**Comment by johndpope** [comment]
overfitted. 
can talk all day about tweaking architecture - 
the longer it trained - the better the results 
conversley - i did some POC work here - which shortened training time for convergence
https://github.com/johndpope/IMF/issues/39


i have some private rust code for nvc - but struggled to get the model into a portable format. 
i think only option is tensorflow js - onnx couldn't cut it.
https://github.com/google-ai-edge/ai-edge-torch/issues/305

**Comment by samsara-ku** [comment]
@HongchengZhao Hi, I'm currently interested in this project.

How many epochs should be passed to achieve those quality of results?
